vbfml/scripts/err_analysis

```diff
@@ -378,5 +384,67 @@ def evaluate(ctx, input_dir: str, sequence_type: str) -> None:
     model.evaluate(sequence)
 
 
+@cli.command()
+@click.pass_context
+@click.option(
+    "-i", "--input-dir", required=True, multiple=True, help="files to compare"
+)
+@click.option(
+    "-n",
+    "--number-epochs",
+    required=False,
+    type=int,
+    default=5,
+    help="The number of last epochs considered in the statistics",
+)
+def compare_hist(ctx, input_dir: str, number_epochs: int, save_name: str) -> None:
+    """
+    Compare the statistics of the history of different models.
+    """
+
+    for dir in input_dir:
+        cache_file = pjoin(dir, "history.pkl")
+        assert os.path.exists(cache_file), f"Cannot find file: {cache_file}"
+        with open(cache_file, "rb") as f:
+            hist = pickle.load(f)
+
+        x = hist["x_val_categorical_accuracy"][-number_epochs:]
+        acc = hist["y_val_categorical_accuracy"][-number_epochs:]
+        loss_tr = hist["y_loss"][-number_epochs:]
+        loss_val = hist["y_val_loss"][-number_epochs:]
+
+        plt.figure(1)
+        plt.plot(x, acc, ".-", markersize=15, label=os.path.basename(dir))
+        plt.figure(2)
+        plt.plot(x, loss_tr, ".-", markersize=15, label=os.path.basename(dir))
+        plt.figure(3)
+        plt.plot(x, loss_val, ".-", markersize=15, label=os.path.basename(dir))
+
+    outdir = pjoin(os.path.dirname(input_dir[0]), "comparison" + save_name)
+    if not os.path.exists(outdir):
+        os.makedirs(outdir)
+
+    plt.figure(1)
+    plt.legend()
+    plt.title("comparison validation accuracy")
+    plt.xlabel("epochs")
+    plt.ylabel("accuracy")
+    plt.savefig(outdir + "/comparison_val_acc" + save_name + ".pdf")
```

Comment:
For these filenames, you can use f-strings + `pjoin` function for better readability, e.g.: 
`plt.savefig( pjoin(outdir, f'comparison_val_acc{save_name}.pdf') )`

Reply:
done